.release-notes/4060.md

```diff
@@ -0,0 +1,7 @@
+## Allow overriding the return type of FFI functions
+
+On the sync call for March 1st, we decided to re-add the ability to specify a return type for FFI function calls. This was previously removed by RFC 68 on the grounds that, with FFI declarations being mandatory, specifying a return type would be redundant.
```

Comment:
Is there a reason for calling out the sync call? that seems like extra, unneeded info. if it is because people could go to listen to  get more context, i think we should try to provide full context in the release notes.

I'd like to change these release notes to show something from red or someone else that was brought up in zulip that was a completely valid use case and wasn't possible due to the change.

Reply:
Sure, my bad, I blindly copy-pasted the commit message into the release notes. Let me rework it a bit.